composer.json

```
@@ -0,0 +1,16 @@
+{
+    "name": "onedesign/atomic-deploy",
+    "description": "A script to easily handle atomic deployments in a Linux environment.",
+    "type": "library",
+    "version": "0.0.1",
+    "bin": [
+        "bin/atomic-deploy"
```

Can we rename this to just `deploy`?
```suggestion
        "bin/deploy"
```